src/foraging_gui/Foraging.py

```diff
@@ -392,7 +392,10 @@ def _ReconnectBonsai(self):
         logging.info('attempting to restart bonsai')
         self.InitializeBonsaiSuccessfully=0       
         self._InitializeBonsai()
+        if self.InitializeBonsaiSuccessfully ==1 and hasattr(self, 'GeneratedTrials'):
```

Comment:
What is this doing?

Reply:
I added a comment, but its not showing up in the Files Changed viewer. 

There is a bug #166, where if trials have been generated, and then bonsai reconnects, when new trials are generated the trial loop hangs. I cannot resolve the bug, so I am warning users to start a new session.